targets/web/src/applyAnimatedValues.ts

```diff
@@ -70,7 +74,12 @@ export function applyAnimatedValues(instance: Instance, props: Lookup) {
 
   // Apply DOM attributes
   names.forEach((name, i) => {
-    instance.setAttribute(name, values[i])
+    const value = values[i]
+    if (value !== void 0) {
+      instance.setAttribute(name, value)
+    } else {
+      instance.removeAttribute(name)
+    }
   })
 
   if (className !== void 0) {
```

Comment:
should we apply the same logic of removing to all the props here? Just wondering the rationale behind the ones you've changed at the moment.